src/main/java/org/llorllale/cactoos/matchers/MatchesBefore.java

```diff
@@ -40,7 +44,7 @@
  *       "must run in 5 seconds maximum",
  *       new TextOf("test"),
  *       new MatchesBefore(
- *           5000,
+ *           5000Open,
```

Comment:
@andreoss I think there is a typo here :)

Reply:
@victornoel Fixed